dependencies/pip/dev_requirements.txt

```diff
@@ -685,4 +688,3 @@ yubico-client==1.13.0
 
 # The following packages are considered to be unsafe in a requirements file:
 # setuptools
-backports-zoneinfo==0.2.1; python_version < '3.9'
```

Comment:
Do not remove `backports-zoneinfo==0.2.1; python_version < '3.9'`  until we support Python3.8

Reply:
Added it back (sorry, pip removed that and I should have asked about it).

Reply:
Handled in #5125